src/games/strategy/triplea/ui/MapPanel.java

```diff
@@ -462,10 +462,8 @@ public void setTopLeft(final int x, final int y) {
   }
 
   // this one is useful for screenshots
-  @Override
-  public void print(final Graphics g) {
+  public void drawMapImage(final Graphics g) {
     final Graphics2D g2d = (Graphics2D) g;
-    super.print(g2d);
```

Comment:
In order to display a progress indicator, I had to move the core logic for saving a screenshot to a background thread.  Of all the code in this method, the only bit that appeared to require running on the EDT was the call to `super.print()`.  I compared screenshots generated with this line both present and removed, and both screenshots were identical.  Therefore, I concluded this call had no effect on the final image and could be safely removed.

I renamed the method, not only because I removed the `super.print()` call, but because I didn't want to imply the method must be called from the EDT.  **Please advise if the method name should be reverted or changed to something else.**

In retrospect, adding Javadocs for this method, including a statement that the method is safe to call from a non-EDT thread, would probably be a good idea.